pom.xml

```diff
@@ -287,7 +288,13 @@
 				<artifactId>logback-classic</artifactId>
 				<version>${version.qos.logback}</version>
 			</dependency>
-
+			
+			<dependency>
+	        	<groupId>org.projectlombok</groupId>
+	        	<artifactId>lombok</artifactId>
+	        	<version>${version.lombok}</version>
+	        	<scope>provided</scope>
```

Comment:
Why provided?

Reply:
lombok is compile-time only.  See https://projectlombok.org/mavenrepo/